package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "roles-subgraph",
+  "version": "1.0.0",
+  "author": "orium.network",
+  "license": "MIT",
+  "description": "A set of subgraphs (one for each network) for Orium Network dapps",
+  "main": "index.js",
```

I don't think we have that file `index.js`
```suggestion
  "main": "index.js",
```